advanced/Scripts/query.sh

```diff
@@ -107,9 +107,35 @@ scanDatabaseTable() {
     # behavior. The "ESCAPE '\'" clause specifies that an underscore preceded by an '\' should be matched
     # as a literal underscore character. We pretreat the $domain variable accordingly to escape underscores.
     if [[ "${table}" == "gravity" ]]; then
+        local abpquerystr, abpfound, abpentry, searchstr
```

Comment:
```suggestion
        local abpquerystr abpfound abpentry searchstr
```

No commas needed here, invalid bash syntax

```
root@docker-pihole-pi4:/# pihole -q 007angels.com
 Match found in https://raw.githubusercontent.com/StevenBlack/hosts/master/hosts:
   007angels.com
   www.007angels.com
 Match found in https://small.oisd.nl/:
   ||007angels.com^
root@docker-pihole-pi4:/# pihole -q test.007angels.com
 Match found in https://small.oisd.nl/:
   ||007angels.com^
```